Assets/Scripts/Minigames/CowardMiniGame/RipcordBehavior.cs

```diff
@@ -119,25 +185,30 @@ IEnumerator FindPreview()
     }
     void OnTriggerEnter(Collider col)
     {
-        if(col.gameObject.tag == "Release" && _hasReleased == true)
+        if (col.gameObject.tag == "Release" && _hasReleased == true)
         {
             _numReleased++;
             GameObject.Destroy(col.gameObject);
         }
 
-        if(col.gameObject.tag == "Player")
+        if (col.gameObject.tag == "Player")
```

Comment:
Prefer checking of the obj has the player controller instead of a string comparison